package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
`git rm` this file (see my [slack post](https://berkeley-codebase.slack.com/archives/C01ALD3S275/p1602882988006200))